swing-lib/src/main/java/org/triplea/swing/JLabelBuilder.java

```diff
@@ -33,9 +33,14 @@
   private Integer borderSize;
   private int biggerFont;
   private Color foregroundColor;
+  private boolean bold;
 
   public JLabelBuilder() {}
 
+  public JLabelBuilder(final String text) {
+    this.text = text;
+  }
```

Comment:
I think this constructor is a bit inconsistent, it somewhat breaks the builder pattern IMO